packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1401,25 +1367,6 @@ class RenderPhysicalModel extends _RenderCustomClip<RRect> {
     return null;
   }
 
-  @override
-  bool hitTest(HitTestResult result, { Offset position }) {
-    if (_clipper != null) {
-      _updateClip();
-      assert(_clip != null);
-      if (!_clip.contains(position))
-        return false;
-    }
-    return super.hitTest(result, position: position);
-  }
-
-  static final Paint _defaultPaint = new Paint();
-  static final Paint _transparentPaint = new Paint()..color = const Color(0x00000000);
-
-  // On Fuchsia, the system compositor is responsible for drawing shadows
-  // for physical model layers with non-zero elevation.
-  @override
-  bool get alwaysNeedsCompositing => _elevation != 0.0 && defaultTargetPlatform == TargetPlatform.fuchsia;
-
   @override
   void paint(PaintingContext context, Offset offset) {
```

Comment:
Who calls _shape.paint()?

Reply:
Aimilarly to RenderPhysicalModel, this currently doesn't paint the border, but just clips and casts shadow.
This is intended to be used by the Material widget, which currently doesn't paint any border... (it is using RenderPhysicalModel to clip itself to a shape though)

Reply:
we should probably have Material paint the border (doesn't have to be in this PR though).